README.md

```diff
@@ -40,7 +40,27 @@ If the MDI file is found and is correct the following actions are taken:
   
 ARGUMENTS
```

Comment:
Please, also rename the directory `multiDeposit` to `multideposit`. For some reason it is still in mixed case.